src/main/webapp/app/exercises/modeling/manage/modeling-exercise-update.component.html

```diff
@@ -33,28 +33,34 @@ <h3 jhiTranslate="artemisApp.exercise.sections.general">General Information</h3>
                 <jhi-category-selector [categories]="exerciseCategories" [existingCategories]="existingCategories" (selectedCategories)="updateCategories($event)" />
             </div>
         }
-        <h3 jhiTranslate="artemisApp.exercise.sections.mode">Mode</h3>
+        <h3 jhiTranslate="artemisApp.exercise.sections.mode" id="artemisApp.exercise.sections.mode">Mode</h3>
```

Comment:
I generally agree, but that would require a data overhead to be passed to the validation bar, since the id is needed there to navigate through the page. Since the translation string should be unique among the headlines here, this works as id and simplifies the code there.

Reply:
Ah ok that makes sense then!